app/grants/templates/grants/explorer.html

```diff
@@ -55,6 +55,15 @@
       background: none !important;
     }
 
+    .alert-fixed-top {
+      position: fixed;
+      top: 80px;
+      right: 0;
+      z-index: 1030;
+      width: 450px;
+      max-width: 95%;
+    }
+
```

Comment:
would it make sense to share this style in a different css given that it's used in the other page as well?

Reply:
Yeah, this could live in `app/assets/v2/scss/gc-utilities.scss` @chibie

Reply:
done ✅  https://github.com/gitcoinco/web/pull/10344/commits/7b242bb5726a03ac3ad3ddc87442d94326b8e563